modules/gitops-release-notes-1-13-2.adoc

```diff
@@ -0,0 +1,51 @@
+// Module included in the following assembly:
+//
+// * release_notes/gitops-release-notes.adoc
+
+:_mod-docs-content-type: REFERENCE
+
+[id="gitops-release-notes-1-13-2_{context}"]
+= Release notes for {gitops-title} 1.13.2
+
+{gitops-title} 1.13.2 is now available on {OCP} 4.12, 4.13, 4.14, 4.15, and 4.16.
+
+[id="errata-updates-1-13.2_{context}"]
+== Errata updates
+
+[id="rhsa-XXXX:NNNN-gitops-1-13-2-security-update-advisory_{context}"]
+=== RHSA-XXXX:NNNN - {gitops-title} 1.13.2 security update advisory
+
+Issued: 2024-10-29
+
+The list of security fixes that are included in this release is documented in the following advisory:
+
+* link:https://access.redhat.com/errata/RHSA-XXXX:NNNN[RHSA-XXXX:NNNN]
```

Comment:
Same question about the ID numbers as above.

Reply:
I have answered this question in an earlier comment.